letsencrypt/cli.py

```diff
@@ -679,7 +679,7 @@ def create_parser(plugins, args):
 # For now unfortunately this constant just needs to match the code below;
 # there isn't an elegant way to autogenerate it in time.
 VERBS = ["run", "auth", "install", "revoke", "rollback", "config_changes",
-         "plugins"]
+         "plugins", "--help"]
```

Comment:
I agree that it's not a verb and is a dirty hack but try running `letsencrypt --help [topic]` without it in the list. It doesn't work.

With that said, `letsencrypt -h [topic]` doesn't work either.